README.md

```diff
@@ -25,7 +25,9 @@ This crate provides several optional components which can be enabled via [Cargo
 
 - `process`: Enable [process metrics](https://prometheus.io/docs/instrumenting/writing_clientlibs/#process-metrics) support.
 
-- `push`: Enable [push metrics](https://prometheus.io/docs/instrumenting/pushing/) support.
+- `push`: Enable [push metrics](https://prometheus.io/docs/instrumenting/pushing/) support with blocking network calls. Incompatible with `push-async`.
```

Comment:
> Incompatible with `push-async`.

Keep in mind that Rust features are additive, thus no feature should be incompatible with another one.

https://github.com/rust-lang/cargo/issues/4328

Reply:
Just an idea: For simplicity, how about leaving push-async alone and remove push, considering that from async to sync is easy (just execute an executor block_on). Discussion is welcome!

Reply:
I'd be happy to do that! It just depends on what the maintainers of this project want -- I'm not sure how they balance breaking changes vs. updating to the latest and greatest. All my Rust projects are using async, so, selfishly, I'd be fine with that.

Reply:
Actually I just changed my PR. Now the async functions are exported in the `push` feature. I've addressed @mxinden's concern about instantiating lazy_statics unnecessarily by moving their instantiation to their call site. Now you'll only instantiate a reqwest client if you actually use it to push metrics. I think is an improvement!